yarn.lock

```diff
@@ -1570,14 +1577,10 @@ js-yaml@^3.9.1:
     argparse "^1.0.7"
     esprima "^4.0.0"
 
-jsbn@0.1.0:
+jsbn@0.1.0, jsbn@~0.1.0:
```

Comment:
I imagine you could get away with using a caret version here to use the latest version of jsbn